.utility/deploy_docs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    pip install --user virtualenv
+    wget -O generate_copy.sh https://raw.githubusercontent.com/fossasia/yaydoc/master/generate.sh
```

Comment:
The generate.sh scripts deletes itself at the end. So I'm downloading a seperate copy. I could have just created a copy of the script. I'll do that.